src/components/calcite-link/readme.md

```diff
@@ -0,0 +1,50 @@
+# calcite-button
```

Comment:
this should probably say calcite-link, though

Reply:
D'oh